gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -61,16 +65,30 @@ public class AndroidCommandPluginExtension {
     // prefer system property over direct setting to enable commandline arguments
     @Memoized
     def getDeviceId() {
-        if (System.properties['deviceId'])
+        if (System.properties['deviceId']) {
             return System.properties['deviceId']
-        if (deviceId instanceof Closure)
+        } else if (deviceId instanceof Closure) {
             return deviceId.call()
+        }
         deviceId ?: defaultDeviceId()
     }
 
-    // prefer system property over direct setting to enable commandline arguments
```

Comment:
I think this is still a valuable comment. Why did you remove it? Is it obvious that the ordering here is related to overriding properties on the command line?